samples/sqlservices/package.json

```diff
@@ -82,7 +82,114 @@
           ]
         }
       }
-    ]
+    ],
+    "connectionProvider": {
+      "providerId": "testProvider",
```

Comment:
Apparently Notebook kernel code requires the provider ID to be upper case : https://github.com/microsoft/azuredatastudio/blob/main/src/sql/workbench/services/notebook/browser/models/notebookModel.ts#L1021

@chlafreniere @VasuBhog Can we consider this a bug? It's not documented anywhere and I can't think of a good reason why we'd want the provider IDs to be case sensitive

Reply:
Hmm can you confirm @Charles-Gagnon? The jupyter provider isn't upper case: https://github.com/microsoft/azuredatastudio/blob/main/extensions/notebook/package.json#L614

Reply:
This is for connection providers (I haven't verified myself but @JustinMDotNet ran into this when working on the log analytics extension)

Reply:
I can discuss with Justin to see what the issue is/verify the issue and if there is a bug address it to make sure provider IDs don't have to be case sensitive.

Reply:
@VasuBhog any update?

Reply:
I changed this for Kusto to test but it seems the connection management is also case sensitive and was not listing the Kusto connections available. I will be using Justin's extension instead to verify. Will try and get it confirmed by EOD tomorrow.

Reply:
Make sure you check whether the connection management specifically requires upper case or if it's just case sensitive.

Reply:
After some testing, was able to see that the connection management is case-sensitive and that is likely the issue I faced with Kusto. I have been looking into the uppercase notebooks issue with the log analytics and did find that there seems to be an issue with the casing on the providerID upon loading the connection. I will investigate further as it seems to affect both kernels dropdown and attach to depending on if we cast it to uppercase later.